script/copy_deduplicate

```diff
@@ -32,10 +32,15 @@ WITH
   FROM
     `{live_table}`
   WHERE
-    DATE(submission_timestamp) = DATE(@start_time)
+    DATE(submission_timestamp) >= DATE_SUB(
+        DATE(@start_time),
+        INTERVAL @num_preceding_days DAY
+    )
     AND submission_timestamp < @end_time
   GROUP BY
-    document_id),
+    document_id
+  HAVING
+    submission_timestamp >= @start_time),
```

Comment:
It took me a while to understand this HAVING clause. It might be easier to read through this query if we name the column `MIN(submission_timestamp) AS first_submission_timestamp`, so that we're referencing `first_submission_timestamp` here. Then we'd rename it back to `submission_timestamp` in the final `SELECT`. I suppose that introduces some small risk of a name collision, though.

Reply:
how about just adding this comment:
```suggestion
    -- HAVING applies *after* aggregation, so remove document_ids first seen before @start_time
    submission_timestamp >= @start_time),
```